integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/e2e/ssh/SshSudoCommandTest.java

```diff
@@ -0,0 +1,137 @@
+package com.sequenceiq.it.cloudbreak.testcase.e2e.ssh;
+
+import java.util.Set;
+
+import javax.inject.Inject;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.Test;
+
+import com.sequenceiq.cloudbreak.auth.crn.RegionAwareInternalCrnGeneratorFactory;
+import com.sequenceiq.freeipa.api.v1.operation.model.OperationState;
+import com.sequenceiq.it.cloudbreak.client.FreeIpaTestClient;
+import com.sequenceiq.it.cloudbreak.client.UmsTestClient;
+import com.sequenceiq.it.cloudbreak.context.Description;
+import com.sequenceiq.it.cloudbreak.context.TestContext;
+import com.sequenceiq.it.cloudbreak.dto.environment.EnvironmentTestDto;
+import com.sequenceiq.it.cloudbreak.dto.freeipa.FreeIpaTestDto;
+import com.sequenceiq.it.cloudbreak.dto.freeipa.FreeIpaUserSyncTestDto;
+import com.sequenceiq.it.cloudbreak.dto.ums.UmsTestDto;
+import com.sequenceiq.it.cloudbreak.testcase.e2e.sdx.PreconditionSdxE2ETest;
+import com.sequenceiq.it.cloudbreak.util.ssh.action.SshSudoCommandActions;
+
+public class SshSudoCommandTest extends PreconditionSdxE2ETest {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(SshSudoCommandTest.class);
+
+    private static final String LIST_RULES_FLAG = "-l";
+
+    private static final String CHANGE_USER_TO_ROOT_COMMAND = "su";
+
+    private static final String WORKLOAD_PASSWORD = "Admin@123";
+
+    @Inject
+    private FreeIpaTestClient freeIpaTestClient;
+
+    @Inject
+    private UmsTestClient umsTestClient;
+
+    @Inject
+    private RegionAwareInternalCrnGeneratorFactory regionAwareInternalCrnGeneratorFactory;
+
+    @Inject
+    private SshSudoCommandActions sshSudoCommandActions;
+
+    @AfterMethod(alwaysRun = true)
+    @Override
+    public void tearDown(Object[] data) {
```

Comment:
we won't need this method. role assignments should be cleaned up after calling `notifyResourceDeleted` call to UMS during environment deletion. Also to prevent accidental error regarding this call during test (test is aborted, UMS not deleting assignments, etc.), we have a job on our jenkins, `cloudbreak-ums-user-assignments-cleanup` which is cleaning up assignments of the test users periodically if there is no running ums test job.

Reply:
Removed. Thanks